lib/dialyzer/test/options1_SUITE_data/src/compiler/compile.erl

```diff
@@ -228,11 +228,15 @@ os_process_size() ->
     case os:type() of
 	{unix, sunos} ->
 	    Size = os:cmd("ps -o vsz -p " ++ os:getpid() ++ " | tail -1"),
-	    list_to_integer(lib:nonl(Size));
+	    list_to_integer(nonl(Size));
 	_ ->
 	    0
     end.
 
+nonl([10]) -> [];
```

Comment:
How about `"\n"`?

Reply:
Or $\n to keep the square brackets.
Actually I’d suggest lists:droplast(Size). Surely that OS call always returns a newline.

Reply:
I don't like to move code and do changes at the same time. But I could add a commit that cleans up this ugly stuff. $\n is the right way.